### PR TITLE
BIP Full-RBF deployment schedule

### PR DESCRIPTION
**Pull request by petertodd**

### CODE REVIEW

**Comment by luke-jr** [comment]
RBF is policy and is therefore n/a to the BIP process.

**Comment by petertodd** [comment]
@luke-jr Not all BIP's are about consensus-critical things, e.g. your own [BIP19](https://github.com/bitcoin/bips/blob/master/bip-0019.mediawiki)

File: bip-full-rbf-deadline.mediawiki [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+Full-RBF has significant efficiency advantages(2) over alternatives such as
+FSS-RBF and Child-Pays-For-Parent for a wide variety of common transaction
+patterns such as fee-bumping and multiple sequential payments, as well as smart
+contract protocols such as payment channels and auctions. Miner support would
+let the wider Bitcoin community use the blockchain more efficiently, supporting
+more transactions per second in less blockchain space.
+
+While full-RBF does allow users to "undo" transactions after they have been
+sent, the ability of decentralized wallets to protect users from double-spends
+has proven to be near-zero.(3) Centralized services have had some success in
```
Comment (dabura667) [inline bip-full-rbf-deadline.mediawiki]: Elaborate on how it is proven to be near-zero. This is an important point that deserves its own section imho, just just a reference to a mail (you sent, by the way) on the mailing list.

Even copy-pasting the contents of your mail + formatting would be much better.

Comment (petertodd) [inline bip-full-rbf-deadline.mediawiki]: I should do up a full writeup on this and post it to reddit...

**Comment by petertodd** [comment]
closing, no consensus

**Pull request closed without merging**